crates/turborepo-lib/src/commands/mod.rs

```diff
@@ -51,6 +52,18 @@ impl CommandBase {
         Ok(())
     }
 
+    // NOTE: This deletes the repo config file. It does *not* remove the
+    // `RepoConfig` struct from `CommandBase`. This is fine because we
+    // currently do not have any commands that delete the repo config file
+    // and then attempt to read from it.
+    pub fn delete_repo_config(&mut self) -> Result<()> {
```

Comment:
Since we're just deleting the file I think being explicit about that in the name would be good. (Is there a reason we aren't clearing the config?)
```suggestion
    pub fn delete_repo_config_file(&self) -> Result<()> {
```

Reply:
Basically we'd need to turn the `OnceCell` into a `RefCell` because we want the ability to borrow it and replace the value, while still providing an immutable API.